creativecloud/features/interactive-components/slider-tray/slider-tray.js

```diff
@@ -173,7 +187,39 @@ function sliderEvent(media, layer) {
   });
 }
 
-function uploadImage(media, layer) {
+function cssToPhotoshop(imgObj, adjustment, value) {
+  const unitValue = convertToUnit(adjustment, value, CSSRanges);
+  imgObj[adjustment] = convertFromUnit(adjustment, unitValue, PsRanges);
+}
+
+function convertToUnit(adjustment, value, ranges) {
```

Comment:
convertToUnit => convertToPSUnit

Reply:
This is not PS value, instead a conversion in between.